src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -43,17 +43,24 @@
 
 
 /**
- * Import single-sample GVCFs into GenomicsDB before joint genotyping
+ * Import single-sample GVCFs into GenomicsDB before joint genotyping.
  *
- * The current GATK4 Best Practices for SNP and INDEL Calling use GenomicsDBImport to merge GVCFs from multiple samples, thus
+ * <p>The GATK4 Best Practice Workflow for SNP and Indel calling uses GenomicsDBImport to merge GVCFs from multiple samples.
+ * GenomicsDBImport offers the same functionality as CombineGVCFs and comes from the <i>Intel-Broad Center for Genomics</i>.
+ * The datastore transposes sample-centric variant information across genomic loci to make data more accessible to tools and to manual probing.
```

Comment:
There really is no manual probing.  God help you if you start to dig into the subdirectories. The big selling point is that the storage footprint is much smaller than the output of CombineGVCFs.